QModManager/Utility/Dialog.cs

```diff
@@ -121,61 +144,54 @@ private IEnumerator ShowDialogEnumerator()
             }
 
             // Turn the dialog blue if the blue parameter is true
-            Sprite sprite = confirmation.gameObject.GetComponentInChildren<Image>().sprite;
-            if (color == DialogColor.Blue)
+            Sprite = confirmation.gameObject.GetComponentInChildren<Image>().sprite;
```

Comment:
local variables should start lowercase

Reply:
it is no longer local as it gets used in the callback which moved out of that method